scripts/postinstall.js

```diff
@@ -3,19 +3,19 @@
 /* eslint-disable no-undef */
 /* eslint-disable @typescript-eslint/no-require-imports */
 const fs = require('node:fs')
-const { join, relative, dirname } = require('node:path')
+const { join, relative, dirname, resolve } = require('node:path')
 
 if (!process.env.INIT_CWD) return
-// TODO: check if were in a local install
+
 const nodeModules = join(process.env.INIT_CWD, 'node_modules')
-if (!fs.existsSync(nodeModules)) return
 const typesDir = join(nodeModules, '@types')
-if (!fs.existsSync(typesDir)) fs.mkdirSync(typesDir)
+if (!fs.existsSync(typesDir)) fs.mkdirSync(typesDir, {recursive: true})
```

Comment:
```suggestion
// we may have to create node_modules altogether in case of a mono repo
if (!fs.existsSync(typesDir)) fs.mkdirSync(typesDir, {recursive: true})
```